app/domain/data.ts

```diff
@@ -70,6 +70,17 @@ export type GeoData = {
   symbolLayer: SymbolLayer | undefined;
 };
 
+// Extracted for performance reasons.
```

Comment:
Curiosity : What was the problem ? Where was it extracted from ?

Reply:
As here we were fighting for every ms, I wanted to avoid the resolver chain with additional queries, as basically our SearchCube was needed in its full form, so we didn't need to have a way to e.g. only specify themes or labels (see the sparqlEditorUrl, data, sparqlEditorUrl).

<img width="307" alt="Screenshot 2023-10-31 at 09 12 29" src="https://github.com/visualize-admin/visualization-tool/assets/52032047/327d6084-9bb5-4112-9785-815e34324062">

Maybe we didn't get a lot of performance improvement here, but with this + making it a Scalar, we only send one query.

<img width="137" alt="Screenshot 2023-10-31 at 09 15 05" src="https://github.com/visualize-admin/visualization-tool/assets/52032047/2c6644ff-28cc-4540-8e84-e69dbc695014">

Reply:
Understood, thanks !